src/plotting/plotWindow.component.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PlotSettings from './plotSettings.component';
+import Plot from './plot.component';
+import { Box, Grid, Drawer, IconButton, Typography } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings';
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import { AxisSettings, PlotType } from '../app.types';
+
+interface PlotWindowProps {}
+
+const drawerWidth = 300;
+
+const PlotWindow = (props: PlotWindowProps) => {
+  const [plotTitle, setPlotTitle] = React.useState('');
+  const [plotType, setPlotType] = React.useState<PlotType>('scatter');
+  const [XAxisSettings, setXAxisSettings] = React.useState<AxisSettings>({
+    scale: 'time',
+  });
+  const [YAxesSettings, setYAxesSettings] = React.useState<AxisSettings>({
+    scale: 'linear',
+  });
+
+  const [open, setOpen] = React.useState(false);
```

Comment:
There's advantages to both but is it preferable to have the drawer open by default? This mimics the behaviour of eCat and allows the user to easily see what they can do without knowing to find and click the small settings icon in the top left